build.cake

```diff
@@ -104,12 +104,16 @@ Task ("build")
 	// Tagged build might contain the group name to build specifically if no setting was specified
 	if (GIT_BRANCH.StartsWith("refs/tags/") && !BUILD_NAMES.Any()) {
 		var tagName = GIT_BRANCH.Substring(10);
+		Information ("Trying to build tag: {0}", tagName);
+
 		var buildName = tagName;
 		if (tagName.Contains('-'))
-			buildName = tagName.Substring(0, tagName.LastIndexOf('-'));
+			buildName = tagName.Substring(0, tagName.IndexOf('-'));
```

Comment:
This was causing the tags not to build, if there was a preview in it: `NameThing-1.2.3-preview`